code/game/world.dm

```diff
@@ -7,6 +7,7 @@ GLOBAL_LIST_INIT(map_transition_config, MAP_TRANSITION_CONFIG)
 	// Setup all log paths and stamp them with startups
 	SetupLogs()
 	enable_debugger() // Enable the extools debugger
+	TgsNew(minimum_required_security_level = TGS_SECURITY_TRUSTED) // creates a new TGS object
```

Comment:
Don't we have to actually create `new /datum/tgs_event_handler/impl`?

Reply:
The event handler isn't required, but gives you some [notification options](https://github.com/tgstation/tgstation/blob/master/code/datums/tgs_event_handler.dm)